src/gui_common/TweakedColourPicker.cs

```diff
@@ -35,66 +34,20 @@ public partial class TweakedColourPicker : ColorPicker
     private HSlider sliderROrH = null!;
     private HSlider sliderGOrS = null!;
     private HSlider sliderBOrV = null!;
+
+    // Alpha has all 3 elements to fully hide it
     private HSlider sliderA = null!;
+    private Control labelA = null!;
+    private Control spinboxA = null!;
     private Button pickerButton = null!;
-    private CustomCheckBox? hsvCheckBox;
-    private CustomCheckBox rawCheckBox = null!;
-    private Label htmlColourStart = null!;
-    private LineEdit htmlColourEdit = null!;
-    private HSeparator separator = null!;
-    private GridContainer? presetsContainer;
-    private TextureButton? addPresetButton;
 #pragma warning restore CA2213
 
-    private bool hsvButtonEnabled = true;
-    private bool rawButtonEnabled = true;
-    private bool presetsEditable = true;
-    private bool presetsVisible = true;
-
     // TODO:
     private ColorModeType mode = ColorModeType.Rgb;
 
-    private bool pickingColor;
-    private Color colorBeforePicking;
-    private double pickerTimeElapsed;
-
-    private PresetGroupStorage? groupStorage;
-
-    private delegate void AddPresetDelegate(Color colour);
-
-    private delegate void DeletePresetDelegate(Color colour);
-
     [Export]
     public string PresetGroup { get; private set; } = "default";
```

Comment:
This should have a big warning that this property no longer works. I assume this doesn't work as I saw you remove all of the group handling code.

Reply:
Regular comment or summary? The latter one would be more noticable I think.

Reply:
XML comment as this is a field. And also maybe remove the "export" attribute so that no one can set this from the Godot editor and wonder why it doesn't do anything?